somefile.py

```diff
@@ -42,3 +42,19 @@ def run_command(
 def do_thing():
     print 'thinger'
     print 'thing 2'
+
+import os, re
```

Comment:
E305 expected 2 blank lines after class or function definition, found 1
E402 module level import not at top of file
F811 redefinition of unused 'os' from line 3
F401 're' imported but unused
E401 multiple imports on one line